web-ui/src/api/checkins.js

```diff
@@ -3,6 +3,15 @@ import { resolve } from "./api.js";
 const checkinsUrl = "/services/check-ins";
 const checkinsNoteUrl = "/services/checkin-notes";
 const checkinsPrivateNoteUrl = "/services/private-notes";
+const backgroundInformationUrl = "/services/background-information";
+
+export const updateBackgroundInformation = async (cookie) => {
+  return resolve({
+    url:backgroundInformationUrl,
```

Comment:
If you merge the changes from develop in, I believe you will see that there is now a required field, 'emailAddress' which should also be sent through to the backend to retrieve data. However, it looks like the only available way currently in the background information controller to get info is by Id. This will need to change.